packages/swing-store/src/snapStore.js

```diff
@@ -1,8 +1,9 @@
 // @ts-check
 import { createHash } from 'crypto';
-import { finished as finishedCallback } from 'stream';
+import { finished as finishedCallback, Readable } from 'stream';
 import { promisify } from 'util';
 import { createGzip, createGunzip } from 'zlib';
+import { buffer } from 'stream/consumers';
```

Comment:
This utility was added in Node 16.7. We still officially support Node 14 right now, and it may actually be the default.